### PR TITLE
Fix/448 loading screen size

### PR DESCRIPTION
**Pull request by JohnAGill**
issue: [#448](https://github.com/project-vagabond/covid-green-app/issues/448)

### CODE REVIEW

**Comment by floridemai** [comment]
@JohnAGill Instead of doing all this shenanigans should we use an png for the DOH logo? Actually "re-use" them from here:  https://github.com/project-vagabond/covid-green-app/tree/nys/ios/Assets.xcassets/StateLogo.imageset

**Review by AlanSl — APPROVED** [review]
Can you please remove the commented out section? We've got it in git history and this PR if we need to go back to it.

After that, LGTM, works on different size devices in the simulator

File: src/components/views/loading.tsx
Comment (AlanSl) [inline src/components/views/loading.tsx]: Please remove this before merge as above

File: src/components/views/loading.tsx
Comment (AlanSl) [inline src/components/views/loading.tsx]: Check lint - isn't `Platform` unused now?

**Comment by AlanSl** [comment]
@floridemai FYI the PNG doesn't look great on large high-res Android devices (e.g. this is on a Note 8, open the image full size and you'll see it's slightly blurry and looks a bit small on the screen but it's not a major issue); but if the PNG is necessary to avoid changes on iOS, that's probably more important.

Screenshot_20200918-142003_COVID Alert NY

**Review by AlanSl — CHANGES_REQUESTED** [review]
Putting this in build 43 so don't merge until build 42 is done (and comments, unused variables etc are cleaned up)

**Comment by floridemai** [comment]
> @floridemai FYI the PNG doesn't look great on large high-res Android devices (e.g. this is on a Note 8, open the image full size and you'll see it's slightly blurry and looks a bit small on the screen but it's not a major issue); but if the PNG is necessary to avoid changes on iOS, that's probably more important.

Did you try 1x 2x 3x version (something like https://github.com/project-vagabond/covid-green-app/tree/nys/src/assets/images/callback) as that should work just fine and I think is the exact same way iOS does it on launch s screen (see my prev post). https://reactnative.dev/docs/images
TBH Now i'm wondering what happens with the svgs we use on different screens. share button for example. I think it remains the same everywhere.

We did the width trick for images to cover the entire screen here and there but i'm not sure we can say for sure that will match all launch screens on all devices.

Anyway, feel free to merge if you tested on a couple of them and works ;)

**Comment by floridemai** [comment]
@AlanSl @JohnAGill Ups, I just was that John updated to using PNG. Do you mind asking @helloantoine to provide you 1x 2x 3x versions?

**Comment by JohnAGill** [comment]
Made the above changes and added 2x and 3x images. Also adjusted the logo on the ios splash screen as it didn't match the android version or the pngs

**Review by AlanSl — APPROVED** [review]
Code is clean, looks good on high-res Android, seems to still be fine on iOS simulator, I'm happy

Screenshot_20200917-151152_COVID Alert NY

**Pull request merged**
